src/App.js

```diff
@@ -89,13 +90,22 @@ class App extends React.Component {
               </NavDropdown>
             </Nav>
             <Form inline>
-              <FormControl type="text" placeholder="Search" className="mr-sm-2" />
+              <FormControl value={this.state.searchText}  onChange={(e) => this.handleSearchChange(e)} type="text" placeholder="Search" className="mr-sm-2" />
               <Button variant="outline-success">Search</Button>
             </Form>
           </Navbar.Collapse>
         </Navbar>
     )
   }
+  handleSearchChange = (event) => {
```

Comment:
Wait can you double check that this works with the filter by category feature?

Reply:
Ok I'm testing now.